samgeo/samgeo.py

```diff
@@ -251,12 +256,11 @@ def save_masks(
             dtype = np.uint16
         else:
             dtype = np.uint32
-
         # Generate a mask of objects with unique values
         if unique:
             # Sort the masks by area in ascending order
```

Comment:
"descending order" since you switched the sorting order.
What difference does it make?

Reply:
If the masks are sorted in ascending order by area, the larger ones are at the end of the list. When visualising the masks, the smaller ones were hidden by the larger ones. This caused problems when vectorising.